(MODULES-4369) Use DISM to install iis

wilson208 opened the pull request:
This replaces using puppet-windowsfeature module which is not compatible with Powershell 2.

This change uses puppetlabs-dism module to install IIS in the spec_helper_acceptance.rb file.

jpogran commented on spec/spec_helper_acceptance.rb:
```diff
@@ -29,13 +29,13 @@
-        on(agents, puppet("module install puppet/windowsfeature"))
+        on(agents, puppet("module install puppetlabs/dism"))
         pp=<<-EOS
-    $iis_features = ['Web-WebServer','Web-Scripting-Tools']
```
We need both of these

jpogran commented on spec/spec_helper_acceptance.rb:
```diff
@@ -29,13 +29,13 @@
+          dism { 'IIS-WebServer':
+            ensure    => present,
+            norestart => true,
+            all       => true
```
While this will install the tools, it's overkill. We don't need the extra features, and it will add extra time to the tests

Commit:
(MODULES-4369) Use DISM to install IIS and IIS Management Tools

This replaces using puppet-windowsfeature module which is not compatible with PS2. This change uses puppetlabs-dism module to install IIS in the spec_helper_acceptance.rb file.

wilson208 commented:
@jpogran Should be sorted now. Tested by running iis_application_pool tests and compared times with the old method of installing IIS using puppet-windowsfeature and the new DISM way and got similar times with bot, so I believe this is good to go if you are happy with it.
